### ISSUE TITLE
Can't run from CLI

### ISSUE THREAD

**Issue by CoalaJoe** [libretro/lutro-pong#15]
![image](https://cloud.githubusercontent.com/assets/5689499/23570814/4c1c156a-0066-11e7-99d1-0ace78c86e47.png)
Got this error. Any idea?

**Comment by RobLoach** [comment]
@CoalaJoe Try loading `main.lua`?

Could also try compiling https://github.com/libretro/libretro-lutro and loading the test directory?

I've run into segfaults with the latest Lutro before as well.

**Comment by CoalaJoe** [comment]
Seems like lutro does not work on OSX

**Comment by RobLoach** [comment]
Yeah, I had been getting the Segfault irregularily on Linux as well. There was a HTTP request system that was merged that somewhat broke it sometimes for me when running the tests.

**Comment by CoalaJoe** [comment]
What a pitty. Development on Lutro looks pretty dead.

**Comment by RobLoach** [comment]
@CoalaJoe Let's tackle that over at https://github.com/libretro/libretro-lutro/issues/65
